suite-native/transactions/src/components/TransactionDetail/TransactionDetailInputsSheet.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+
+import { Icon } from '@trezor/icons/src';
+import { WalletAccountTransaction } from '@suite-common/wallet-types/src';
+import { Box, Card, Text, VStack } from '@suite-native/atoms/src';
+import { useFormatters } from '@suite-common/formatters/src';
+import { formatNetworkAmount } from '@suite-common/wallet-utils';
+
+import { TransactionDetailSheet } from './TransactionDetailSheet';
+
+type TransactionAddressAmountProps = {
+    address: string;
+    amount: string;
+};
+const TransactionAddressAmount = ({ address, amount }: TransactionAddressAmountProps) => (
+    <Box>
+        <Text variant="hint" color="gray1000">
+            {address}
+        </Text>
+        <Text variant="label" color="gray600">
+            {amount}
+        </Text>
+    </Box>
+);
+
+type TransactionDetailInputsSheetProps = {
+    isVisible: boolean;
+    transaction: WalletAccountTransaction;
+    handleSheetVisibility: () => void;
+};
+export const TransactionDetailInputsSheet = ({
+    isVisible,
+    handleSheetVisibility,
+    transaction,
+}: TransactionDetailInputsSheetProps) => {
+    const { CryptoAmountFormatter } = useFormatters();
+
+    const txInputs = transaction.details.vin;
+    const txOutputs = transaction.details.vin;
+    const formatInputAddress = (address: string) =>
+        `${address.slice(0, 13)} ... ${address.slice(-6, -1)}`;
```

Comment:
same here if it works with `ellipsizeMode` prop

Reply:
Same as above. With ellipsizeMode you are not in control of how many characters are hidden. In this specific case, the design asks to show more characters at the beginning than at the end of the string. I believe EllipsizeMode does not allow this variability.

Reply:
This logic will be handled in formatter and I think it's just how the guys typed it. Don't see any logical reasons behind number of characters. 

Just use ellipsize for now and we will make this right when formatters get implemented